app/realtime.php

```diff
@@ -27,6 +28,40 @@
 
 Runtime::enableCoroutine(SWOOLE_HOOK_ALL);
```

Comment:
@TorstenDittmann @Meldiron I think it would be great to have the ability to register a error callback on the WebSocket abstraction library like we have in utopia/fw

Reply:
No argument against it, I am all for it 👍🏻

Reply:
Implemented `error` method on utopia-php/websocket to allow registering erro handlers.